### DISCUSSION TITLE
Customise month and year selection in day picker

### DISCUSSION THREAD

**Discussion by princeseth** [gpbl/react-day-picker#2355] [Support]
Hello, Is there any way we can customise month and year selection similar to below. Thanks

![image](https://github.com/user-attachments/assets/d3d2da5a-d248-4720-9187-682fb5a502fb)

![image](https://github.com/user-attachments/assets/bef8af52-fd88-4aa7-9b43-b7cdb04ef5ee)

**Comment by princeseth** [comment]
Still looking for help....😀

**Comment by gpbl** [comment]
@princeseth DayPicker doesn't support that kind of layout for changing the month. However you could create something similar yourself using a custom `Nav` component and controlling the month with the `month`/`setMonth` prop.
